CITATION.cff

```diff
@@ -0,0 +1,10 @@
+cff-version: 1.2.0
+title: 'cuQuantum'
+message: 'If you use this software, please cite it as below.'
+authors:
+  - name: "The cuQuantum development team"
```

Comment:
Note: According to the schema, the `name` key is acceptable. Pandas also did this.
https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#authors